packages/ckeditor5-fullscreen/src/fullscreenediting.ts

```diff
@@ -55,15 +55,16 @@ export default class FullscreenEditing extends Plugin {
 		this.editor.keystrokes.set( 'Ctrl+Shift+F', ( evt, cancel ) => {
 			this.editor.execute( 'toggleFullscreen' );
 
-			// On non-Chromium browsers, the editor view is not blurred properly after moving the editable,
-			// even though the `document.activeElement` is changed. Hence we need to blur the view manually.
+			// On non-Chromium browsers, the editor view and toolbar are not blurred properly after moving the editable,
+			// even though the `document.activeElement` is changed. Hence we need to blur them manually.
 			// Fixes https://github.com/ckeditor/ckeditor5/issues/18250 and https://github.com/ckeditor/ckeditor5/issues/18247.
 			if ( !env.isBlink ) {
 				this.editor.editing.view.document.isFocused = false;
+				this.editor.ui.view.toolbar!.focusTracker.focusedElement = null;
 			}
 
-			this.editor.editing.view.focus();
 			this.editor.editing.view.scrollToTheSelection();
+			this.editor.editing.view.focus();
```

Comment:
If the order here is important maybe it makes sense to mention why in a comment?

Reply:
Actually it doesn't, I changed it in the meantime during testing but it can be both ways - which I think is still worth a comment.